src/AlignedRangeIndexSetBuilders.cpp

```diff
@@ -3,7 +3,7 @@
  *
  * \file
  *
- * \brief   Implementation file for aligned-range index set builder methods.
+ * \brief   Header file for various index set builder methods.
```

Comment:
Not a header.

Reply:
Thanks. I fixed this and the one above. I'm still working on this. I'm going to put in a test for one of the index set builder methods which is still useful, potentially.